matplotlibrc.template

```diff
@@ -387,33 +387,32 @@ backend      : $TEMPLATE_BACKEND
 
 
 ### GRIDS
-#grid.color       :   b0b0b0 # grid color
+#grid.color       :   b0b0b0    # grid color
 #grid.linestyle   :   -         # solid
 #grid.linewidth   :   1.0       # in points
 #grid.alpha       :   1.0       # transparency, between 0.0 and 1.0
 
 ### Legend
-#legend.fancybox      : True  # if True, use a rounded box for the
-                              # legend, else a rectangle
 #legend.loc           : best
-#legend.isaxes        : True
```

Comment:
Correct.  I don't know how long it has been gone, but it is inactive, and marked as such in rcsetup.  It would make no sense for it to be an rcParam; it is entirely internal to legend.  It is a flag to tell whether it is a figure legend or an axes legend.
Probably the validator for it should emit a warning.